scalafix/build.sbt

```
@@ -47,6 +47,14 @@ def scio11(version: String) =
     "com.spotify" %% "scio-avro"
   ).map(_ % version)
 
+def scio12(version: String) =
+  List(
+    "com.spotify" %% "scio-core",
+    "com.spotify" %% "scio-avro",
+    "com.spotify" %% "scio-google-cloud-platform",
+    "com.spotify" %% "scio-extra"
+  ).map(_ % version)
```

🎨 imo we shouldn't parameterize version when the method is already called `scio12`:

```suggestion
lazy val scio12 =
  List(
    "com.spotify" %% "scio-core",
    "com.spotify" %% "scio-avro",
    "com.spotify" %% "scio-google-cloud-platform",
    "com.spotify" %% "scio-extra"
  ).map(_ % Scio.`0.12`)
```